test/e2e/requirements.txt

```diff
@@ -8,6 +8,7 @@ cachetools==4.2.0
 certifi==2020.12.5
 chardet==3.0.4
 execnet==1.7.1
+filelock==3.0.12
```

Comment:
unused, AFAICT.

Reply:
Yes. `pytest-xdist` (the tool we use to multiprocess the tests) does not support session scoped fixtures by default. I had a working model of a workaround for this, using `filelock` and `pickle` but `pytest` had some funky race conditions when using `yield` with these fixtures. I have the code stored in a commit in my local branch if any other service wants to take a stab at it.